xml/templates/schema.tpl

```diff
@@ -10,7 +10,7 @@
 -- {$generated}
 --{if $database.comment} {$database.comment}{/if}
 
-{include file="drop.tpl" no_license=TRUE}
+{include file="drop.tpl" isOutputLicense=FALSE}
```

Comment:
So this is fun: Compare capital FALSE with lower-case false:

`cv ev "$out = CRM_Core_Smarty::singleton()->fetch('string:{assign var=x value=FALSE}{if $x === false}it is false{else}well this is unexpected{/if}'); echo $out;"`

Outputs `well this is unexpected`. If you play further you can see it treats capital FALSE as a string literal.

`cv ev "$out = CRM_Core_Smarty::singleton()->fetch('string:{assign var=x value=false}{if $x === false}it is false{else}well this is unexpected{/if}'); echo $out;"`

Outputs `it is false`, i.e. the lower case one gets treated as a proper boolean.

Reply:
argh

Reply:
updated